tests/rbd_mirror/test-image-delete-from-secondary.py

```diff
@@ -48,8 +48,8 @@ def run(**kw):
             mirror2,
             poolname=poolname,
             imagename=imagename,
-            imagesize=config.get("imagesize", "1G"),
```

Comment:
Any reason we are removing default value >
does it make sense to add default value in utils module definition as a default value for argument?

Reply:
revert back the changes.